DigitalLearningSolutions.Web/Helpers/GenericSortingHelper.cs

```diff
@@ -163,6 +163,13 @@ public class CourseDelegatesSortByOption : Enumeration
             nameof(CourseDelegate.PassRate)
         );
 
+        public static readonly CourseDelegatesSortByOption CandidateNumber = new CourseDelegatesSortByOption(
+            7,
+            nameof(CandidateNumber),
+            "Delegate ID",

```

Comment:
Confirmed by KW on 7th Feb, that it should say "Delegate ID" for this candidate number option.